packages/mui-icons-material/lib/esm/index.js

```diff
@@ -3,16 +3,31 @@ export { default as AbcOutlined } from './AbcOutlined';
 export { default as AbcRounded } from './AbcRounded';
 export { default as AbcSharp } from './AbcSharp';
 export { default as AbcTwoTone } from './AbcTwoTone';
+export { default as AcUnit } from './AcUnit';
```

Comment:
I've seen a similar change before. It looks like there are different ordering rules coming into play. I wonder if this has something to do with the OS or node version used to run the script. 
@chao813 may I ask what OS and node version are you using?

Reply:
@michaldudak 
Node: v16.13.0
OS: macOS

Reply:
Apparently, the order returned by glob is different on a Mac than on Windows. We'll address this in a separate PR. Could you, for now, remove `packages/mui-icons-material/lib/esm/index.js` and `packages/mui-icons-material/lib/index.js` from this PR, so they don't create unnecessary noise?

Reply:
Done, removed those two files from this PR. How did you want to address this in a separate PR?